custom/abt/reports/expressions.py

```diff
@@ -113,9 +113,9 @@ def _get_language(cls, item):
 
     @classmethod
     def _get_warning(cls, spec, item):
-        default = spec.get("warning", "")
+        default = unicode(spec.get("warning", u""))
         if cls._get_language(item) == "fra":
-            return spec.get("french", default)
+            return unicode(spec.get("french", default))
```

Comment:
Curious what problem you were trying to fix here. I'm surprised if `unicode(...)` fixes anything since you have not specified an encoding to use to perform the conversion (the default is ASCII).

Adding `u` to the default arg (`u""`) above is probably correct, and may be the thing that fixed the error in this case.